BFWControls/Modules/Carousel/Controller/CarouselViewController.swift

```diff
@@ -184,13 +196,22 @@ open class CarouselViewController: UICollectionViewController {
         addPageControl()
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        if shouldBounce {
+            showBounceWhenViewLoaded()
+        }
+    }
+    
     open override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
         // Resize cell size to fit collectionView if bounds change.
         if collectionViewSize != collectionView?.bounds.size {
             collectionViewSize = collectionView?.bounds.size
             collectionView?.reloadData()
-            scroll(toPage: pageControl.currentPage, animated: false)
+            if !shouldBounce {
+                scroll(toPage: pageControl.currentPage, animated: false)
+            }
```

Comment:
If we add shouldBounce = false in the showBounce() function then it shouldn't  be an issue.